src/styles/variables.scss

```diff
@@ -0,0 +1,5 @@
+$main-field: #fff;
+$main-accent: #060b35;
+$secondary: #616070;
+$moyo-accent: #00acdc;
+$main-bord: #f3f3f3;
```

Comment:
Now everything is named well, but it is better to also indicate the type of variable in the name, because such names can store, for example, the size or style of the text: `main-field-color`, `secondary-color`.